liquibase-standard/src/main/java/liquibase/snapshot/jvm/ColumnSnapshotGenerator.java

```diff
@@ -501,6 +501,10 @@ protected DataType readDataType(CachedRow columnMetadataResultSet, Column column
                 );
             }
 
+            if (database instanceof MySQLDatabase) {
+                type.setColumnSize(((MySQLDatabase) database).getFSPFromTimeType(columnSize, Types.TIMESTAMP));
```

Comment:
We need to add a null check for `columnSize` here. That's being reported as an alert from our CodeQL scan. Could you please do that?

Thanks,
Daniel.